crates/virtio-queue/src/chain.rs

```diff
@@ -105,9 +105,7 @@ where
         }
 
         // Check the target indirect descriptor table is correctly aligned.
-        if desc.addr().raw_value() & (VRING_DESC_ALIGN_SIZE as u64 - 1) != 0
-            || desc.len() & (VRING_DESC_ALIGN_SIZE - 1) != 0
-        {
+        if desc.len() & (VRING_DESC_ALIGN_SIZE - 1) != 0 {
```

Comment:
I note in the kernel the structures specifically point out this is pre-1.0 behaviour:

```
/* Alignment requirements for vring elements.
 * When using pre-virtio 1.0 layout, these fall out naturally.
 */
#define VRING_AVAIL_ALIGN_SIZE 2
#define VRING_USED_ALIGN_SIZE 4
#define VRING_DESC_ALIGN_SIZE 16
```

However we need to fix the comment as well because we don't check alignment - really all you are doing is checking the descriptor length is a multiple of VRING_DESC_ALIGN_SIZE. This is now a poorly named constant because we really just want to show we have an integer number of struct vring_descs.

Reply:
Thanks for reviewing, I have update the comments, could you please help to check it, thanks.